File: airbyte-commons-protocol/src/main/java/io/airbyte/commons/protocol/transform_models/FieldTransformType.java
/*
 * Copyright (c) 2020-2024 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.commons.protocol.transform_models;

/**
 * Types of transformations possible for a field.
 */
public enum FieldTransformType {
  ADD_FIELD,
  REMOVE_FIELD,
  UPDATE_FIELD_SCHEMA
}

File: airbyte-commons-protocol/src/main/kotlin/io/airbyte/commons/protocol/transformmodels/FieldTransformType.kt
/*
 * Copyright (c) 2020-2025 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.commons.protocol.transformmodels

/**
 * Types of transformations possible for a field.
 */
enum class FieldTransformType {
  ADD_FIELD,
  REMOVE_FIELD,
  UPDATE_FIELD_SCHEMA,
}
